src/accordion/accordion.ts

```diff
@@ -145,6 +154,23 @@ export interface NgbPanelChangeEvent {
   preventDefault: () => void;
 }
 
+/**
+ * An event emitted right before toggling an accordion panel visibility changed, once the animation has been completed.
+ */
```

Comment:
Please, re-word this documentation. I can't manage to understand it as it is.

Reply:
Yeah +1 from me, I haven't left any comments yet on this because I think it will go away if we change the event API

Reply:
Thanks, I just realized that I started to changed a pasted sentence, but without finishing it !
 Anyway, I will remove this event in favor of 'shown' and 'hidden' ones (as Bootstrap for the collapse)